fn/io_test.go

```diff
@@ -0,0 +1,67 @@
+package fn
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	data1 = "The network is robust in its unstructured simplicity."
+	data2 = "Programming today is a race between software engineers " +
```

Comment:
nice quote:)

Reply:
The quote was defined in the previous PR attemps, it was copied from [go ioutil write test](https://github.com/golang/go/blob/d34e2ad19442bed0f49c1de5a84602a741077a50/src/io/ioutil/ioutil_test.go#L48).

I think I will replace it with another Satoshi Whitepaper quote to make it more related to LND.